picard.wdl

```diff
@@ -608,4 +610,35 @@ task SortVcf {
         dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.",
                       category: "advanced"}
     }
-}
\ No newline at end of file
+}
+
+task RenameSample {
+    input {
+        String dockerImage = "quay.io/biocontainers/picard:2.19.0--0"
+        File inputVcf
+        String outputPath
+        String newSampleName
+        Int memory = 8
```

Comment:
This should be a string

Reply:
But there is memory multiplier 
ceil(memory * memoryMultiplier)